python/ray/dashboard/res/main.js

```diff
@@ -0,0 +1,263 @@
+let dashboard = new Vue({
+    el: "#dashboard",
+    data: {
+        now: (new Date()).getTime() / 1000,
+        shown: {},
+        error: "loading...",
+        last_update: undefined,
+        clients: undefined,
+        totals: undefined,
+        tasks: undefined,
+        ray_config: undefined,
+    },
+    methods: {
+        updateNodeInfo() {
+            axios.get("/api/node_info").then(function (resp) {
+                setTimeout(dashboard.updateNodeInfo, 500);
```

Comment:
How about we don't auto refresh by default?

Reply:
Any specific reason? Pretty much all of my use of the dashboard relies on this. A checkbox that disables it maybe?

Reply:
I'm thinking about developers mainly. It was harder to figure out what was happening on the web console due to the large number of requests, and also on the server.

Would be nice to be static by default, with a link to `#autoRefresh`